models/index.js

```diff
@@ -0,0 +1,24 @@
+const { Model } = require('sequelize');
+
+module.exports = (sequelize, DataTypes) => {
+    class User extends Model {
+      static associate(models) {
```

Comment:
Make sure to add a what associate models go here so it does not pull null or remove this part and add associations with the sequelize helpers hasMany, belongsTo, etc inside of the models index.js